.rubocop.yml

```diff
@@ -18,6 +18,12 @@ AllCops:
     - 'truffleruby/target/classes/**/*'
     - 'mxbuild/**/*'
 
+Lint/AssignmentInCondition:
+  Description: Don't use assignment in conditions.
+  StyleGuide: "#safe-assignment-in-condition"
+  Enabled: true
+  AllowSafeAssignment: true
+
```

Comment:
I don't see any value in this change, it's just adding noise:
```diff
 -      while c = getc
 +      while (c = getc)
          yield c.ord
        end
```
(Rubocop should make a difference between loops and conditions at least)
But even in conditions the `(` are the wrong fix:
```diff
 -        if val = @stream.getbyte
 +        if (val = @stream.getbyte)
            return val
          end
```
should really be:
```ruby
          val = @stream.getbyte
          return val if val
```
which I find acceptable although less readable.
It has the advantage of not having side effects (getbyte) in a conditional expression.